### PR TITLE
VR > Event date (end same as start)

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/133307
🎥 https://www.loom.com/share/c7bc8fc176e24f749da6b26bd25a6e76

### CODE REVIEW

File: src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php [hunk 1]
```diff
@@ -9,13 +9,15 @@
...
+ * @version TBD
  *
  * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
  *
  * @see tribe_get_event() For the format of the event object.
  */
-$time_format = tribe_get_time_format();
+
+$time_format      = tribe_get_time_format();
+$display_end_date = $event->dates->start->format( 'H:i' ) !== $event->dates->end->format( 'H:i' );
```
Comment (paulmskim) [inline src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php]: I think we should be using `$event->dates->start_display->format( 'H:i' )` here instead? Is there an equivalent for `end_display`? @lucatume ?

File: src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php [hunk 2]
```diff
@@ -26,12 +28,14 @@ class="tribe-events-calendar-month__calendar-event-datetime-featured tribe-commo
...
-		<?php echo esc_html( $event->dates->end->format( $time_format ) ) ?>
+	<time datetime="<?php echo esc_attr( $event->dates->start->format( 'H:i' ) ); ?>">
+		<?php echo esc_html( $event->dates->start->format( $time_format ) ); ?>
 	</time>
+	<?php if ( $display_end_date ) : ?>
+		<span class="tribe-events-calendar-month__calendar-event-datetime-separator"> - </span>
+		<time datetime="<?php echo esc_attr($event->dates->end->format( 'H:i' ) ); ?>">
+			<?php echo esc_html( $event->dates->end->format( $time_format ) ); ?>
+		</time>
+	<?php endif; ?>
```
Comment (paulmskim) [inline src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php]: `start_display`?

File: src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php (hunk 1 above)
Comment (lucatume) [reply src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php]: There is `$event->dates->end_display`.

File: src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php (hunk 2 above)
Comment (lucatume) [reply src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php]: Whenever we display an event date, we should use `$event->dates->(start|end)_display`.

**Comment by juanfra** [comment]
@paulmskim this is ready for a new review. Thanks!

**Pull request merged**